### DISCUSSION TITLE
Fixing Documentation / Article linkage

### DISCUSSION THREAD

**Discussion by kstingel** [leokhoa/laragon#694] [Q&A]
@Pen-y-Fan, @leokhoa  I'm starting a project fixing broken `Help` linkage within the `Interface` and `laragon.org/Doc` pages

Primarily for my own use so I can better use the software, but once I have completed the task, I plan to make it available via Github

Since you have been doing such a wonderful job of filling in the blanks where `forum.laragon.org` content is missing, may I have your permission to incorporate your help guides into my project?

It is my hope that gathering all the available guides into one concise location will assist @leokhoa in maintaining the API/Docs for future interations of the software

**_Project Goals_**
* Gather complete articles from this Discussion, and within the existing Code, the current Docs, and Medium stories into one source for (cross) reference and reuse
* if permitted, I will reproduce existing content into a `Jetbrains Writerside` project so pages can be reused and modified 
   to reflect future iterations - each 'destination' will be it's own instance so releases can be easily tweaked and repacked with each iteration
* update any older images with screenshots from current (6.0.0) release
* create my own fork and replace broken linkage in the current `master` branch with working links to existing replacement content

**Comment by Pen-y-Fan** [comment]
Thanks, @kstingel, 

You are free to reproduce any of my help guides or blogs, my blog is [Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/deed.en), so you can reuse, I only ask for some attribution.

Regarding **Jetbrains Writerside**,  you are free to copy to any format,  I recommend an open standard like Markdown. All my blog posts are written in Markdown, for example, as is this discussion board.

My thoughts are the legacy forum posts, which have links built into Laragon, could be reproduced as either:

1. Discussion articles, tagged "Laragon's Articles & How-tos". 
2. [Wiki page](https://github.com/leokhoa/laragon/wiki) (which would need further investigation), that way community members could keep them up to date. 

Either option would keep them in one place, @leokhoa could update the Laragon links to either, in which case a Wiki page may be better as the URL would not need to change each each release.

I'm happy to discuss.

**Reply by Pen-y-Fan** [reply]
I used web.archive.org (thanks @josealgardataborges for the idea & link) to re-create the original post for updating PHP, just as an example, this post would need to be updated for PHP 8.3 to make sense now. 

https://github.com/leokhoa/laragon/wiki/%5BTutorial%5D-How-to-add-another-PHP-version-(PHP-7.3)

- What are your thoughts?  
- Are you/other community members able to edit the Wiki?

**Reply by kstingel** [reply]
I'm learning Writerside as I go, so I don't know what the structure of it's Git repo will look like, 
my intention is to have top level folders for each source, then place my material into organised folders beneath that

no image at present as I'm setting up the scaffold as I go, I'll edit and add an example image later :)

I hadn't realised the `Wiki` section was live - just checked and I can add/edit pages there, so as I recreate the old documentation, I'll create any missing pages in there

I've created my own [fork](https://github.com/kstingel/laragon) of this repo, once I've finished importing the upstream Wiki content, I'll create new pages there and submit the forked Wiki into `Show and Tell` Discussion so it's easily located :)

**Reply by kstingel** [reply]
I can't contibute financially, but I have the time available to sort out some of the Documentation issues
possibly slow going initially as R/L takes time too, but once the daily grind stuff is dealt with, I can 'work' on cleanup/revival here

**Comment by kstingel** [comment]
persuant to fixing missing links and creating help resources,

I just added a new page to Wiki :) - [Updating-ngrok](https://github.com/leokhoa/laragon/wiki/Updating-ngrok) 
still needs usage information added, but I didn't want to include that since I myself don't know how to use it

**Comment by kstingel** [comment]
since I don't have (repo) tweak access here - and I'm not asking for it
I have created new `'Discussion' Categories` on [my fork](https://github.com/kstingel/laragon) where I can post ideas and Guides prior to inclusion here :)
have sent out Collaboration invites
